client/post-editor/editor-revisions-list/item.jsx

```diff
@@ -70,7 +70,9 @@ class EditorRevisionsListItem extends PureComponent {
 
 					{ added === 0 &&
 						removed === 0 && (
-							<span className="editor-revisions-list__minor-changes">{ translate( 'minor' ) }</span>
+							<span className="editor-revisions-list__minor-changes">
+								{ translate( 'minor', { context: 'post revisions: minor changes' } ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **10** times:
`translate( 'Minor: ' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?